packages/core/src/index.ts

```diff
@@ -41,15 +37,18 @@ export async function clearTestMailQueue() {
   }
 
   try {
-    await fs.unlinkSync(TMP_TEST_FILE);
+    fs.unlinkSync(TMP_TEST_FILE);
   } catch (e: any) {
     if (e.code === "ENOENT") return; // file does not exist
     throw e;
   }
 }
 
 export function buildSendMail(options: BuildSendMailOptions) {
-  const testMode = process.env.TEST || process.env.NODE_ENV === "test";
+  const testMode =
+    process.env.TEST ||
+    process.env.NODE_ENV === "test" ||
+    process.env.MAILING_CI;
```

Comment:
@psugihara the reason this doesn't work is we do `import { buildSendMail } from "mailing-core"` (the node_modules package) in the generator templates.  So a change to core will not be reflected when in dev mode.  Do you think we should:
a) cherry-pick this into mailing-core and release it? seems harmless
b) get the generator templates to reference packages/core and search/replace them to the current line during build process

Reply:
Doing A seems better if we have to do one of those. I would have thought that pre-construct would handle yalcing mailing-core though.

Reply:
true ```>mailing ls -l node_modules/mailing-core
lrwxr-xr-x  1 alexfarrill  staff  16 Oct 12 13:57 node_modules/mailing-core -> ../packages/core``` maybe I just have to yarn build it before?  will try that

Reply:
OK that worked, I guess you have to build because it is using the code in the dist folder

Reply:
I get it, in core's package.json we tell it that main is `"dist/mailing-core.cjs.js",` so that's where it looks when it does module resolution